web/README.md

```diff
@@ -0,0 +1,23 @@
+The `web` subdirectory contains the React based modern web UI, related assets such as fonts and images, translations, and some miscellaneous legacy bits and pieces.  
+For the legacy web UI, please see the `java` subdirectory.  
+
+The `web` subdirectory consists of roughly the following main chunks:  
+
+ - `web/po`: Translations.
+ - `web/html/javascript`: Legacy scripts, most of them global. Over time we're slowly trying to sunset these piece by piece.  
+ - `web/html/src/branding`: Branding assets, such as stylesheet sources, fonts, images etc.
+ - `web/html/src/build`: Build tooling for the web UI.
+ - `web/html/src/components`, `web/html/src/core`, `web/html/src/manager`, `web/html/src/utils`: Source code for the web UI.
+ - `web/html/dist`: Output directory for the frontend build, do not check this directory in nor modify it directly, your changes will be overwritten by the next build.  
```

Comment:
Should we mention that `web/modules` is obsolete, scheduled for removal, and must not be used?

Reply:
Check with other developers, but I am 99.99% sure it's not needed.

Reply:
I'm not familar with those parts, hopefully someone else can chime in on that. If it's scheduled for removal, is there a ticket for it?

Reply:
I think there was a card a couple of years ago about a cleanup for stuff from spacewalk, but probably it's closed alreeady and we'd need a new one.

For now, at least we should document.

@mcalmer am I right about `web/modules`?